### PR TITLE
Add Comment basic implementation.

### PR DESCRIPTION
**Pull request by marciopd**
_Add comment_ implemented with a new component _AddCommentPanel_. _Comment_ component name refactored to _CommentPanel_.

### CODE REVIEW

File: src/main/java/com/vaadin/training/bugrap/ui/component/CommentPanel.java [hunk 1]
```diff
@@ -2,27 +2,33 @@
...
-public class Comment extends CustomComponent {
+public class CommentPanel extends CustomComponent {
 
 	private static final long serialVersionUID = -7113423443746306442L;
 
 	private static final String AUTHOR_TEMPLATE = "%s (%s)";
 	private static final String ANONYMOUS = "Anonymous";
 
-	private final CommentDesign commentDesign;
+	private final CommentPanelDesign commentDesign;
```
Comment (Pasadena) [inline src/main/java/com/vaadin/training/bugrap/ui/component/CommentPanel.java]: This seems a bit redundant. Perhaps better way would be to make CommentPanel extend CommentPanelDesign and extend from CustomComponent when there is no design-class available

File: src/main/java/com/vaadin/training/bugrap/ui/component/AddCommentPanel.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+	private void initCancelButton() {
+		cancelButton.addClickListener(event -> {
+			getApplicationModel().reset();
+			clearInputFields();
+		});
+	}
+
+	private void initDoneButton() {
+		doneButton.addClickListener(event -> {
```
Comment (Pasadena) [inline src/main/java/com/vaadin/training/bugrap/ui/component/AddCommentPanel.java]: The done-button could be disabled when comment-field is empty

File: src/main/java/com/vaadin/training/bugrap/ui/model/ReportPanelModel.java [hunk 3]
```diff
@@ -201,4 +202,17 @@ public void setSelectReportById(final String reportIdParam) {
 		selectedReports.add(report);
 		UIEventBus.getInstance().publish(new ReportsSelectedEvent(selectedReports));
 	}
+
+	public void addComment() {
```
Comment (Pasadena) [inline src/main/java/com/vaadin/training/bugrap/ui/model/ReportPanelModel.java]: Is this needed?

Comment (marciopd) [reply src/main/java/com/vaadin/training/bugrap/ui/model/ReportPanelModel.java]: Dead code. Will be removed.

File: src/main/java/com/vaadin/training/bugrap/ui/component/AddCommentPanel.java (hunk 2 above)
Comment (marciopd) [reply src/main/java/com/vaadin/training/bugrap/ui/component/AddCommentPanel.java]: Ok!

File: src/main/java/com/vaadin/training/bugrap/ui/component/CommentPanel.java (hunk 1 above)
Comment (marciopd) [reply src/main/java/com/vaadin/training/bugrap/ui/component/CommentPanel.java]: Ok!

**Review by Pasadena — APPROVED** [review]

**Pull request merged**